pkg/sql/testdata/mysql_test_expected_diffs_on_information_schema.json

```diff
@@ -2,9 +2,9 @@
   "pgVersion": "8.0.23",
   "diffSummary": {
     "TotalTables": 79,
-    "TotalColumns": 224,
-    "MissingTables": 57,
-    "MissingColumns": 71,
+    "TotalColumns": 280,
+    "MissingTables": 0,
+    "MissingColumns": 9,
```

Comment:
what are the 9 missing columns? maybe we could add those too.

Reply:
I found an issue from this mysql, which seemed outdated, these are the actual values:

```
"diffSummary": {
    "TotalTables": 48,
    "TotalColumns": 280,
    "MissingTables": 21,
    "MissingColumns": 80,
    "DatatypeMismatches": 0
  }
```